inference-engine/tests/functional/shared_test_classes/src/subgraph/multiple_connect_split_concat.cpp

```diff
@@ -0,0 +1,44 @@
+// Copyright (C) 2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "shared_test_classes/subgraph/multiple_connect_split_concat.hpp"
+
+namespace SubgraphTestsDefinitions {
+std::string MultipleConnectSplitConcatTest::getTestCaseName(const testing::TestParamInfo<MultipleConnectSplitConcatParams> &obj) {
+    InferenceEngine::Precision netPrecision;
+    std::string targetDevice;
+    std::map<std::string, std::string> configuration;
+    std::tie(netPrecision, targetDevice, configuration) = obj.param;
+
+    std::ostringstream result;
+    result << "netPRC=" << netPrecision.name() << "_";
+    result << "targetDevice=" << targetDevice << "_";
+    for (auto const &configItem : configuration) {
+        result << "_configItem=" << configItem.first << "_" << configItem.second;
+    }
+    return result.str();
+}
+
+void MultipleConnectSplitConcatTest::SetUp() {
+    InferenceEngine::Precision netPrecision;
+    std::tie(netPrecision, targetDevice, configuration) = this->GetParam();
+    auto ngPrc = FuncTestUtils::PrecisionUtils::convertIE2nGraphPrc(netPrecision);
+
+    auto params = ngraph::builder::makeParams(ngPrc, {{1, 256}});
+    auto relu_start = std::make_shared<ngraph::opset1::Relu>(params[0]);
+    auto split = ngraph::builder::makeSplit(relu_start, ngPrc, 2, 1);
+    auto const_concat = ngraph::builder::makeConstant(ngPrc, {1, 96}, std::vector<float>{0});
+    auto const_concat_2 = ngraph::builder::makeConstant(ngPrc, {1, 96}, std::vector<float>{0});
```

Comment:
After looking deeper the setup should be like this:
split with 1 output
concat with 2 inputs that are using the same split output

Reply:
fixed as recommended